zetaclient/zetacore/tx_test.go

```diff
@@ -106,22 +101,6 @@ func Test_GasPriceMultiplier(t *testing.T) {
 	}
 }
 
-func getHeaderData(t *testing.T) proofs.HeaderData {
-	var header ethtypes.Header
-	file, err := os.Open("../../testutil/testdata/eth_header_18495266.json")
```

Comment:
These testdata might not be needed anymore, we should check

Reply:
it is used by `x/liteclient` and `pkg/proof`